package labs_examples.exception_handling.labs;

/**
 * Exception Handling Exercise 3:
 *
 *      Demonstrate a try/catch/finally block.
 *
 */
//  try{
//      // code that needs to be monitored for errors
//  } catch (Exception e){
//     // handler for Exception object e
//  } finally{
//     // code to execute no matter the result of try/catch
//  }
